package.json

```diff
@@ -16,12 +16,19 @@
     "dev": "yarn workspace @botpress/messaging-server dev",
     "board": "yarn workspace @botpress/messaging-board dev",
     "channels": "yarn workspace @botpress/messaging-channels example",
+    "test:setup": "docker run -p 2345:5432 --rm --name msg_test_postgres -d -e POSTGRES_USER=postgres -e POSTGRES_PASSWORD=postgres -e POSTGRES_DB=postgres postgres && sleep 2",
+    "test:teardown": "docker container stop msg_test_postgres",
     "test": "yarn test:unit && yarn test:int && yarn test:mig && yarn test:e2e && yarn test:sec",
+    "test:pg": "yarn test:unit && yarn test:int:pg && yarn test:mig:pg && yarn test:e2e:pg && yarn test:sec:pg",
```

Comment:
Any reason we can't have test:setup and test:teardown inside this command?

Reply:
I added it at first, but then considering that we don't do it for each sub-command, I thought it would be better to just have to do it all the time. Would you prefer it if those commands were added here?